### PR TITLE
Fixed cast of as_node and as_relationship

### PR DESCRIPTION
**Pull request by wey-gu**
UT added, too

### CODE REVIEW

**Review by Aiee — COMMENTED** [review]
Have you tried the examples? We better add these examples to CI like other SDKs to make sure that the demo code is always valid.

**Comment by wey-gu** [comment]
Tried the simple one and it worked.

True, the example could be added in another pr.

File: requirements/example.in [hunk 1]
```diff
@@ -1 +1,2 @@
 prettytable
+pandas
```
[the remark this answers is not in the document]
Comment (wey-gu) [reply requirements/example.in]: > Please add a new line at the end.

Done

**Comment by wey-gu** [comment]
created issue https://github.com/vesoft-inc/nebula-python/issues/228 for introducing example validation in ci

**Review by Aiee — APPROVED** [review]

**Pull request merged**